src/libstd/io/stdio.rs

```diff
@@ -30,18 +30,27 @@ thread_local! {
 ///
 /// This handle is not synchronized or buffered in any fashion. Constructed via
 /// the `std::io::stdio::stdin_raw` function.
+///
+/// The size of a StdinRaw struct may vary depending on the target operating
```

Comment:
Since these types aren't publicly exported do you think we need to mention their size variance?

Reply:
Oh, if it's not public then yeah, that can be omitted.